### PR TITLE
Additional detection for LeagueClientUx.exe to cover linux users.

### PR DESCRIPTION
**Pull request by Its-Haze**
I tried running lcu-driver on Fedora/Nobara linux. when i play League from Lutris/Wine.
The processes that were looked for did not check "CrBrowserMain" causing me not being able to use the project on Linux.

I verified that it works with this change included on Linux, but i have not tested on Windows.. I doubt that it will cause any issues to include this process.

image

### CODE REVIEW

File: lcu_driver/utils.py [hunk 1]
```diff
@@ -13,9 +13,18 @@ def parse_cmdline_args(cmdline_args) -> Dict[str, str]:
 
 
 def _return_ux_process() -> Generator[Process, None, None]:
-    for process in process_iter():
-        if process.status() == STATUS_ZOMBIE:
-            continue
+    for process in process_iter(attrs=["cmdline"]):
+        try:
```
Comment (sousa-andre) [inline lcu_driver/utils.py:17]: What's the purpose of this try/catch? If such errors occur we should know to fix them (e.g. [known problems](https://github.com/sousa-andre/lcu-driver/tree/master#problems)) and not hide them away

File: lcu_driver/utils.py [hunk 2]
```diff
@@ -13,9 +13,18 @@ def parse_cmdline_args(cmdline_args) -> Dict[str, str]:
...
+                continue
 
-        if process.name() in ['LeagueClientUx.exe', 'LeagueClientUx']:
-            yield process
+            cmdline: List[str] = process.info.get("cmdline", [])
+
+            if process.name() in ["LeagueClientUx.exe", "LeagueClientUx"]:
+                yield process
+
+            if cmdline and cmdline[0].endswith("LeagueClientUx.exe"):
```
Comment (sousa-andre) [inline lcu_driver/utils.py:26]: Are you able to add a small comment before the `if` statement to remember why that condition is needed for my own future reference?

File: lcu_driver/utils.py (hunk 1 above)
Comment (Its-Haze) [reply lcu_driver/utils.py:17]: I tested without it and got no problems. So i'd say it's probably safe to have it like it was before.  (Meaning no try:catch block.)

File: lcu_driver/utils.py (hunk 2 above)
Comment (Its-Haze) [reply lcu_driver/utils.py:26]: Sure i can do that

**Comment by Its-Haze** [comment]
The reason for the single quites turning into double quotes is that i have Black,  setup to auto format my files.. I recommend using it throughout the code base but if that's not preffered i can turn them back to single quotes..

**Comment by sousa-andre** [comment]
Looks good to me. Thank you for taking the time fixing this issue. I really appreciate it.
(also nice profile pic :p)

**Comment by Its-Haze** [comment]
Thanks! :)

**Pull request merged**